Wrappers/Python/cil/framework/data_container.py

```diff
@@ -476,7 +476,7 @@ def multiply(self, other, *args, **kwargs):
     def divide(self, other, *args, **kwargs):
         if hasattr(other, '__container_priority__') and \
            self.__class__.__container_priority__ < other.__class__.__container_priority__:
-            return other.divide(self, *args, **kwargs)
+            return other.power(-1).multiply(self, *args, **kwargs)
```

Comment:
This may save some memory?
```suggestion
            _out = other.divide(self, *args, **kwargs)
            _out.power(-1, out=_out)
            return _out
```